main.js

```diff
@@ -1,4 +1,23 @@
-// Remember your promises
-// Go back to the lesson if you feel lost
-// Read the docs well, they usually have pretty well documented examples
-// Consider making your fetch work with a button
+const img = document.createElement('img');
+const character = document.createElement('p');
+const button = document.querySelector('#random');
+const apiUrl = `https://rickandmortyapi.com/api/character/`;
+const rickAndMorty = () => {
+    for (i = 1; i < 6; i++) {
+        axios({
+            url: apiUrl + [i],
+            method: 'get',
+        })
+        .then(res => {
+            const imgUrl = res.data.image
+            const name = res.data.name
+            img.src = imgUrl;
```

Comment:
```suggestion
           const img = document.createElement('img');
            img.src = imgUrl;
```
You can create image element with each loop so you can display all images in the page.
also you got the name but I think you forget to add it tho the page
```
const para = document.createElement('p')
para.innerText = name
document.body.appendChild(para);
```